### PR TITLE
test(trigger): trigger tablets performance tests

### PR DESCRIPTION
**Pull request by juliayakovlev**
Start triggering tablets tests every third week

### Testing

- [ ]

### PR pre-checks (self review)

- [x] I added the relevant `backport` labels
- [x] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevant to this change (if needed)

### CODE REVIEW

File: jenkins-pipelines/performance/branch-perf-v15/scylla-master/weekly-enterprise-performance-tablets-trigger.xml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+</description>
+  <keepDependencies>false</keepDependencies>
+  <scm class="hudson.scm.NullSCM"/>
+  <canRoam>true</canRoam>
+  <disabled>false</disabled>
+  <blockBuildWhenDownstreamBuilding>false</blockBuildWhenDownstreamBuilding>
+  <blockBuildWhenUpstreamBuilding>false</blockBuildWhenUpstreamBuilding>
+  <triggers>
+    <hudson.triggers.TimerTrigger>
+      <spec>00 6 * * ? 3/3</spec>
```
Comment (fruch) [inline jenkins-pipelines/performance/branch-perf-v15/scylla-master/weekly-enterprise-performance-tablets-trigger.xml:14]: let's try using this:
`0 6 */21 * *`

**Comment by fruch** [comment]
@juliayakovlev let's move the elasticity case into this trigger

File: jenkins-pipelines/performance/branch-perf-v15/scylla-master/weekly-enterprise-performance-tablets-trigger.xml (hunk 1 above)
Comment (juliayakovlev) [reply jenkins-pipelines/performance/branch-perf-v15/scylla-master/weekly-enterprise-performance-tablets-trigger.xml]: done

**Comment by juliayakovlev** [comment]
> @juliayakovlev let's move the elasticity case into this trigger

done

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**